db/re/skill_db.yml

```
@@ -45018,6 +45018,37 @@ Body:
         - Level: 5
           Amount: 78
     Status: WeaponBlock_On
+  - Id: 6518
+    Name: CD_DIVINUS_FLOS
+    Description: Divinus Flos
+    MaxLevel: 5
+    Type: Magic
+    TargetType: Attack
+    DamageFlags:
+      Splash: true
+    Range: 9
+    Hit: Multi_Hit
+    HitCount: 5
```

I guess Hitcount should be negative? 5 hits with a base skillratio of 20000 seems unbalanced. Please confirm (if you know)
```suggestion
    HitCount: -5
```

---

I'll keep it here until i test

---

So this change could be accepted?

---

Yes but it seems that it requires https://github.com/rathena/rathena/pull/9491

---

So i can add it with commented flag MultiHitOnPlants then on #9491 i'll uncomment it.